rep-2004.rst

```diff
@@ -375,54 +306,43 @@ These packages are more lax on documentation, testing, and scope of public API's
 Package Requirements
 ~~~~~~~~~~~~~~~~~~~~
 
-*Note: bullets below that start with [ROS Core], will be the prescription for what we do in the core packages in order to meet the associated requirements*
-
 Requirements to be considered a 'Level 3' package:
 
-* Version Policy:
-
-  * The same as 'Level 1' packages, except:
-
-    * No public API needs to be explicitly declared, though this can make it harder to maintain API and ABI stability
-    * No requirement to keep API/ABI stability within a stable ROS release, but it is recommended still
+#. **Version Policy:**
 
-* Change Control Process:
+   #. The same as 'Level 1' packages, except:
 
-  * Must have all code changes occur through a change request (e.g. pull request, merge request, etc.)
-  * Must have Continuous Integration (CI) policy for all change requests
-  * [ROS Core]:
+      #. *No public API needs to be explicitly declared, though this can make it harder to maintain API and ABI stability*
+      #. *No requirement to keep API/ABI stability within a stable ROS release, but it is recommended still*
 
-    * All changes will go through a pull request
-    * All pull requests will be tested via CI
+#. **Change Control Process:**
 
-* Documentation:
+   #. Must have all code changes occur through a change request (e.g. pull request, merge request, etc.)
+   #. Must have Continuous Integration (CI) policy for all change requests
 
-  * Must have a declared license or set of licenses
-  * Must have a copyright statement in each source file
-  * May have a "quality declaration" document, which declares the quality level and justifies how the package meets each of the requirements
+#. **Documentation:**
 
-    * Must have a section in the repository's ``README`` which contains the "quality declaration" or links to it
-    * May register with a centralized list of 'Level 3' packages, if one exists, to allow for peer review of the claim
+   #. Must have a declared license or set of licenses
+   #. Must have a copyright statement in each source file
+   #. May have a "quality declaration" document, which declares the quality level and justifies how the package meets each of the requirements
```

Comment:
are the "May..." statements the same as the "recommended" statements? for example level 3 `1.i.b`: `No requirement to keep API/ABI stability within a stable ROS release, but it is recommended still`

i mapped all the "may..." sentences to ● in the chart but idk if that's accurate